docs/basics/installation.rst

```diff
@@ -46,41 +91,205 @@ You can find detailed installation instructions on how to do this
-is to enable the `NeuroDebian <http://neuro.debian.net/>`_ repository.
-If you are on a Debian-based system, but do not have the NeuroDebian repository
-enabled, you should very much consider enabling it right now. The above hyperlink links
-to a very easy instruction, and it only requires copy-pasting three lines of code.
+There are two ways to get DataLad on Windows 10: one is within Windows itself,
+the other is using WSL, the Windows Subsystem for Linux. **We** *strongly*
+**recommend the latter.**
 
-The following command installs
-DataLad and all of its software dependencies (including the Git-annex-standalone package):
+Note: Using Windows comes with some downsides.
+In general, DataLad feels a bit sluggish on Windows systems. This is because of
+a range of filesystem issues that also affect the version control system :term:`Git` itself,
+which DataLad relies on. The core functionality of DataLad should work, and you should
+be able to follow the contents covered in this book.
+You will notice, however, that some
+Unix commands displayed in examples may not work given the installation that you
+chose, and that terminal output can look different from what is displayed here.
+If you are a Windows user and want to help improve the handbook for Windows users,
+please `get in touch <https://github.com/datalad-handbook/book/issues/new>`_.
 
-.. code-block:: bash
+.. container:: toggle
 
-   $ sudo apt-get install datalad
+   .. container:: header
 
+      **1) Install within Windows**
 
-OS X
-----
+   Note: This installation method will get you a working version of
+   DataLad, but be aware that many Unix commands shown in the book
+   examples will not work for you, and DataLad-related output might
+   look different from what we can show in this book.
 
-A common way to install packages on OS X is via the
-`homebrew <https://brew.sh/>`_ package manager.
-Git-annex can be installed using the command
+   - **Step 1**: If you haven't, install Python3
 
-.. code-block:: bash
+      - Check for Python in your start menu. If no search results
+        show up, go to https://www.python.org/downloads.
+        The most recent version of Python is on the "Download"
+        button near the top of the page. Click on ``download``,
+        then ``save``.
 
-   $ brew install git-annex
+      - Run the Python installer by double-clicking the downloaded
+        ``python-<version>.exe`` file in your Downloads or clicking
+        on the Python 3.x result in your start menu.
+
+      - **Check the box "Add Python <version> to PATH** at the bottom
+        of the window, and select "Customize installation".
+
+      - Stay with the default options to install all optional features,
+        and additionally tick
+        "Add Python to environment variables" on the second page.
+        Optionally, tick the box
+        "Install for all users" to ensure that other users on the computer
+        are able to use Python.
```

Comment:
Consider trimming the content here in favor of https://docs.python.org/3/using/windows.html (same reason as above, things are complex and can change without us noticing)